packages/lb-utils/src/PointCloudUtils.ts

```diff
@@ -533,6 +534,33 @@ class PointCloudUtils {
       };
     });
   }
+
+  // Get the point to line distance
+  static getDistance(line: [ICoordinate, ICoordinate], p: ICoordinate) {
```

Comment:
You can use `getFootOfPerpendicular` in `lb-annotation/src/utils/MathUtils`. And move this function to `packages/lb-utils/src/PointCloudUtils.ts`